pyvista/jupyter/pv_pythreejs.py

```diff
@@ -422,9 +427,15 @@ def actor_to_mesh(actor, focal_point):
 
         meshes.append(to_surf_mesh(actor, surf, mapper, prop, add_attr))
 
-    elif rep_type == 'Points':
+    if rep_type == 'Points':
```

Comment:
If I understand correctly this change doesn't change anything, because `if rep_type == 'Surface' and has_faces:` and `if rep_type == 'Points':` are mutually exclusive anyway.

Reply:
Correct. I was probably working through the `has_verts` branch and didn't revert this change.